samples/AndroidSample/AndroidSample.csproj

```diff
@@ -10,34 +10,6 @@
     <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.AndroidX.Legacy.Support.V4" Version="1.0.0.23" />
-    <PackageReference Include="Xamarin.AndroidX.AppCompat" Version="1.6.1.7" />
-    <PackageReference Include="Xamarin.AndroidX.Core" Version="1.12.0.4" />
-    <PackageReference Include="Xamarin.AndroidX.Compose.UI" Version="1.6.4" />
-    <PackageReference Include="Xamarin.AndroidX.Compose.Runtime" Version="1.6.4" />
-    <PackageReference Include="Xamarin.AndroidX.Compose.Runtime.RxJava3" Version="1.6.4" />
-    <PackageReference Include="Xamarin.AndroidX.ConstraintLayout" Version="2.1.4.10" />
-    <PackageReference Include="Xamarin.AndroidX.ExifInterface" Version="1.3.7.1" />
-    <PackageReference Include="Xamarin.AndroidX.Fragment" Version="1.6.2.2" />
-    <PackageReference Include="Xamarin.AndroidX.RecyclerView" Version="1.3.2.2" />
-    <PackageReference Include="Xamarin.AndroidX.CardView" Version="1.0.0.25" />
-    <PackageReference Include="Xamarin.AndroidX.GridLayout" Version="1.0.0.23" />
-    <PackageReference Include="Xamarin.AndroidX.Palette" Version="1.0.0.23" />
-    <PackageReference Include="Xamarin.AndroidX.Preference" Version="1.2.1.4" />
-    <PackageReference Include="Xamarin.AndroidX.ViewPager2" Version="1.0.0.25" />
-    <PackageReference Include="Xamarin.AndroidX.WebKit" Version="1.10.0.1" />
-    <PackageReference Include="Xamarin.Google.Android.Material" Version="1.10.0.3" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.Common" Version="2.7.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.7.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData.Core" Version="2.7.0.1" />
-    <PackageReference Include="Xamarin.AndroidX.MultiDex" Version="2.0.1.23" />
-    <PackageReference Include="Xamarin.AndroidX.Browser" Version="1.8.0" />
-    <PackageReference Include="Square.OkHttp3" Version="4.12.0.2" />
-    <PackageReference Include="Xamarin.Kotlin.StdLib" Version="1.9.23" />
-    <PackageReference Include="Xamarin.Kotlin.StdLib.Jdk8" Version="1.9.23" />
-    <PackageReference Include="nventive.binding.kotlinx.serialization.json" Version="1.2.2" />
-    <PackageReference Include="Xamarin.Android.ReactiveX.RxJava3.RxAndroid" Version="3.0.2.8" />
-    <PackageReference Include="Xamarin.Android.ReactiveX.RxJava3.RxJava" Version="3.1.8.3" />

```

Comment:
Should all these dependencies be removed?

Reply:
They are not necessary as the binding project refers to the same packages so transitively the sample project would reference them as well.
Removing them from sample saves us from the effort to always ensure same versions are referenced in both the project...

Reply:
Makes sense, thanks!